src/main/java/mate/academy/service/AuthenticationServiceImpl.java

```diff
@@ -2,11 +2,16 @@
 
 import mate.academy.exception.AuthenticationException;
 import mate.academy.model.User;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
 
 public class AuthenticationServiceImpl implements AuthenticationService {
+    private static final Logger logger = LogManager.getLogger(AuthenticationServiceImpl.class);
+
     @Override
     public User login(String login, String password) throws AuthenticationException {
         //TODO: add corresponding log message about method login was called
+        logger.info("The login method was called. Params: login = {}", login);
```

Comment:
The logging message 'The login method was called. Params: login = {}' is informative, but it might be a security risk to log sensitive information such as login credentials. Consider logging only non-sensitive information or using a more generic message.